projects/fal/src/fal/api.py

```diff
@@ -210,6 +211,8 @@ def wrapper(*remote_args: ArgsT.args, **remote_kwargs: ArgsT.kwargs) -> ReturnT:
                 f"Uncaught user function exception: {str(exc)}"
             ) from exc.with_traceback(tb)
         finally:
+            with suppress(Exception):
+                patch_pickle()
             with suppress(Exception):
                 patch_dill()
```

Comment:
Doesn't hurt anything to patch both, but could consider adding a flag for the wrapper or something.